stdlib/2and3/distutils/log.pyi

```diff
@@ -0,0 +1,28 @@
+from typing import List, Any, Callable, Iterable
+
+DEBUG: int = ...
+INFO: int = ...
+WARN: int = ...
+ERROR: int = ...
+FATAL: int = ...
+
+class Log:
+    def __init__(self, threshold: int = ...) -> None: ...
+    def log(self, level: int, msg: str, *args: Any) -> None: ...
+    def debug(self, msg: str, *args: Any) -> None: ...
+    def info(self, msg: str, *args: Any) -> None: ...
+    def warn(self, msg: str, *args: Any) -> None: ...
+    def error(self, msg: str, *args: Any) -> None: ...
+    def fatal(self, msg: str, *args: Any) -> None: ...
+
+LogFunc = Callable[[str, Iterable[Any]], None]
+
+log: Callable[[int, str, Iterable[Any]], None]
```

Comment:
Why isn't this just `log: LogFunc`?

Reply:
`log` is the more general form, supporting the extra `level: int` parameter.